crates/agglayer-storage/src/stores/state/mod.rs

```diff
@@ -0,0 +1,40 @@
+use std::sync::Arc;
+
+use super::StateReader;
+use crate::{
+    columns::latest_certificate_per_network::LatestSettledCertificatePerNetworkColumn,
+    error::Error, storage::DB, types::NetworkId,
+};
+
+#[cfg(test)]
+mod tests;
+
+/// A logical store for the state.
+pub struct StateStore {
+    db: Arc<DB>,
+}
+
+impl StateStore {
+    pub fn new(db: Arc<DB>) -> Self {
+        Self { db }
+    }
+}
+
+impl StateReader for StateStore {
+    /// Get the active networks.
+    /// Meaning, the networks that have at least one submitted certificate.
+    ///
+    /// Performance: O(n) where n is the number of networks.
+    /// This is because we need to scan all the keys in the
+    /// `last_certificate_per_account` column family.
```

Comment:
I don't find such column, do you mean `latest_settled_certificate_per_network`?

I think the related benchmark `last_certificate_bench.rs` also refer to a different name of this column